deploy.sh

```diff
@@ -2,7 +2,7 @@
 
 rev=$(git rev-parse --short HEAD)
 
-cd target/doc
+cd target/doc || exit 72
```

Comment:
What's magic about "72"?

Reply:
Should be 66 actually.

72 means "a system file hasn't been found", 66 means "a non system file hasn't been found"

That follows libc standard exit codes (and as there is no other standard specific for bash, it's a good way to explain what happens).

Reply:
Are you talking about sysexits.h?  That's not part of libc.  It actually comes from Sendmail.  libc only defines `EXIT_SUCCESS` and `EXIT_FAILURE`.  sysexits.h seems to have fallen out of favor years ago, at least in the BSD world.

http://pubs.opengroup.org/onlinepubs/009695399/basedefs/stdlib.h.html